app/helpers/react_on_rails_helper.rb

```diff
@@ -89,10 +89,18 @@ def react_component(component_name, props = {}, options = {})
     <<-HTML.html_safe
 #{data_from_server_script_tag}
 #{rendered_output}
-#{replay_console(options) ? console_script : ""}
+#{replay_console(options) ? console_script : ''}
     HTML
   end
 
+  def sanitized_props_string(props)
```

Comment:
Next PR.

@samnang 's idea didn't work -- need to find the right thing to escape.